src/UsgsAstroLsSensorModel.cpp

```diff
@@ -193,11 +191,15 @@ void UsgsAstroLsSensorModel::replaceModelState(const std::string &stateString )
    m_halfTime = j["m_halfTime"];
    m_imageFlipFlag = j["m_imageFlipFlag"];
    // Vector = is overloaded so explicit get with type required.
-   m_ephemPts = j["m_ephemPts"].get<std::vector<double>>();
-   m_ephemRates = j["m_ephemRates"].get<std::vector<double>>();
+   std::cerr << "D" << std::endl;
+
+   m_positions = j["m_positions"].get<std::vector<double>>();
+   m_velocities = j["m_velocities"].get<std::vector<double>>();
    m_quaternions = j["m_quaternions"].get<std::vector<double>>();
-   m_parameterVals = j["m_parameterVals"].get<std::vector<double>>();
+   m_currentParameterValue = j["m_currentParameterValue"].get<std::vector<double>>();
    m_covariance = j["m_covariance"].get<std::vector<double>>();
+
+   std::cerr << "Unpacked The State JSON" << std::endl;

```

Comment:
All the debug cerrs should be removed. This can happen here or elsewhere. There are similar debugs in the framer.